api.md

```diff
@@ -2654,65 +2656,222 @@ SELECT * FROM timescaledb_information.data_node;
-| `table_size` |Disk space used by hypertable |
-| `index_size` |Disk space used by indexes|
-| `toast_size` |Disk space of toast tables|
-| `total_size` |Total disk space used by the specified table, including all indexes and TOAST data|
-| `distributed` | (BOOLEAN) Distributed status of the hypertable |
+| `table_schema` | Schema name of the hypertable |
+| `table_name` | Table name of the hypertable |
+| `table_owner` | Owner of the hypertable |
+| `num_dimensions` | Number of dimensions |
+| `num_chunks` | Number of chunks |
+| `compression_enabled` |Is compression enabled on the hypertable?|
+| `is_distributed` | Is the hypertable distributed?|
+| `replication_factor` | Replication factor for a distributed hypertable|
+| `data_nodes` | Nodes on which hypertable is distributed|
+| `tablespaces` |Tablespaces attached to the hypertable |
 
-#### Sample Usage [](timescaledb_information-hypertable-examples)
+#### Sample Usage [](timescaledb_information-hypertables-examples)
 
-Get information about all hypertables.
+Get information about a hypertable.
 
 ```sql
-SELECT * FROM timescaledb_information.hypertable;
+CREATE TABLE dist_table(time timestamptz, device int, temp float);
+SELECT create_distributed_hypertable('dist_table', 'time', 'device', replication_factor => 2);
+
+SELECT * FROM timescaledb_information.hypertables
+  WHERE table_name = 'dist_table';
+
+-[ RECORD 1 ]-------+-----------
+table_schema        | public
+table_name          | dist_table
+owner               | postgres 
+num_dimensions      | 2
+num_chunks          | 3
+compression_enabled | f
+is_distributed      | t
+replication_factor  | 2
+data_nodes          | {node_1, node_2}
+tablespaces         | 
 
- table_schema | table_name | table_owner | num_dimensions | num_chunks | table_size | index_size | toast_size | total_size | distributed
---------------+------------+-------------+----------------+------------+------------+------------+------------+------------+--------------
- public       | metrics    | postgres    |              1 |          5 | 99 MB      | 96 MB      |            | 195 MB     | t
- public       | devices    | postgres    |              1 |          1 | 8192 bytes | 16 kB      |            | 24 kB      | f
-(2 rows)
 ```
 
-Check whether a table is a hypertable.
+## timescaledb_information.dimensions [](timescaledb_information-dimensions)
+
+Get metadata about the dimensions of hypertables, returning one row of metadata for each dimension of a hypertable.  For a time-and-space-partitioned hypertable, for example, two rows of metadata will be returned for the hypertable.
+
+If the dimension is time based, the dimension column has either an integer datatye or a time-based datatype. The `time_interval` column is defined for hypertables that use time-based datatypes (e.g., timestamptz, timestamp, date).  Alternatively, for hypertables that use integer datatypes (e.g., bigint, integer), the `integer_interval` and `integer_now_func` columns are defined.
```

Comment:
Doesn't that first sentence contradict the 2nd and 3rd sentence. Did you mean to say open dimension in the first sentence?

Reply:
I think it is confusing. Let me reword that.